.github/workflows/cicd.yml

```diff
@@ -0,0 +1,45 @@
+---
+# This workflow will install Python dependencies, run tests
+
+name: Run CI/CD
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: ["3.9"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: FedericoCarboni/setup-ffmpeg@v2
+        id: setup-ffmpeg
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Install test dependencies
+        run: |
+          pip install pytest
```

Comment:
Can we reuse the requirements_dev.txt file to install all test/dev requirements.